src/utils/narrow.js

```diff
@@ -134,3 +134,17 @@ export const narrowFromMessage = (message: Message, email: string) => {
 
   return streamNarrow(message.display_recipient);
 };
+export const compareNarrow = (narrow1: Narrow, narrow2: Narrow): boolean => {
```

Comment:
How is this different than just calling `isEqual` from lodash? (which is a deep equality)

Reply:
Yup it does, just to test it works added some test's too!